raiden/tests/scenarios/mfee3_imbalance_fees.yaml

```diff
@@ -0,0 +1,71 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: any
+  services:
+    pfs:
+      url: https://pfs-goerli-with-fee.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_min: 1_000_000
+  balance_fund: 1_000_000
```

Comment:
Can we use the same values as what most other scenarios have been updated to? So `10_000_000_000_000_000_000` for `balance_fund`. `1_000_000_000_000_000_000` deposits and transfers of `1_000_000_000_000_000`? This is in order to make sure the fee calculations are correct for more realistic amounts.

Reply:
👍